modules/virt-configuring-sriov-networks-for-vms.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-defining-an-sriov-network.adoc
+
+// This is similar to the OpenShift `configuring-sr-iov-networks` module,
+// modified for virt context
+
+// Because of an existing issue in go-yaml, the strings 'on' and 'off'
+// are interpreted as booleans, not strings. The SR-IOV admission controller
+// will reject 'spoofCheck' and 'trust' if the values are not strings.
+// So these values must be explicitly quoted in the YAML.
+// https://github.com/go-yaml/yaml/issues/214
+
+[id="virt-configuring-sr-iov-networks-for-vms_{context}"]
+= Configuring SR-IOV additional network for virtual machines
+
+You can configure an additional network that uses SR-IOV hardware by creating a SriovNetwork Custom Resource (CR).
+When you create a SriovNetwork CR, the SR-IOV Operator automatically creates a NetworkAttachmentDefinition CR.
+
+[NOTE]
+=====
+Do not modify or delete a SriovNetwork Custom Resource (CR) if it is attached to any Pods in the `running` state.
+=====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
```

Comment:
Do we mention this in every single module?

Reply:
In general, if the module calls for oc commands, it's included as a prereq